tools/internal_ci/linux/grpc_aws_experiment_remote.sh

```diff
@@ -7,7 +22,5 @@ sudo pip install six
 
 cd grpc
 # build with bazel
-export CXX=clang++
-export CC=clang
-tools/bazel test --config=dbg //test/...
+tools/bazel build --config=opt //test/...
```

Comment:
In my experience, bazel fails to find the toolchain unless explicitly set.

Reply:
It does work just fine: sponge/c88eebb3-e707-4993-80d9-b14a5eed57d3